Probesys/bileto
# misc: Reorganize custom CSS and timeline CSS

## Related issue(s)



https://github.com/Probesys/bileto/issues/229
https://github.com/Probesys/bileto/issues/233

## Changes



- rename stylesheets/pages in stylesheets/custom
- rename list-tickets.css in tickets.css and change the class `.tickets-list__id` to the more generic `.ticket__id` class
- put the ticket anchor in the ticket title
- rename organizations-index.css in organizations.css (moved to components/) and rename the classes
- move the events.css code in timeline.css and rename the classes
- use variables in the timeline to improve the maintainability 

## How to test manually



- go to a tickets list → check everything looks fine
- go to a ticket → check everything looks fine

## Checklist
